flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/enumerator/assigner/SplitAssigner.java

```diff
@@ -34,7 +33,7 @@
  * readers and store all the state into checkpoint.
  */
 @Internal
-public interface SplitAssigner extends Serializable {
```

Comment:
Why do you make this change?

Reply:
`SplitAssigner` would be created dynamically. No need to make it `Serializable`.